### PR TITLE
Fix form fields ax-load issue on spa-mode

### PR DESCRIPTION
**Pull request by akunbeben**
- [x] Changes have been thoroughly tested to not break existing functionality.
- [ ] New functionality has been documented or existing documentation has been updated to reflect changes.
- [x] Visual changes are explained in the PR description using a screenshot/recording of before and after.

As per #9151 fixes the table ax-load issue on spa mode, the same issue also happened in some form fields. 
This PR will fix it.

https://github.com/filamentphp/filament/assets/46495960/acc6b039-3aaa-47a2-8e50-4abf1cf7f671

### CODE REVIEW

**Comment by akunbeben** [comment]
DateTimePicker still has the issue with the displayText etc. I'm working on it

**Comment by zepfietje** [comment]
Thanks for working on this, @akunbeben!
Could you mark this PR as ready for review when you've implemented all pending changes?

**Comment by ArtMin96** [comment]
I have the same issue. Thanks @akunbeben

**Comment by akunbeben** [comment]
> Thanks for working on this, @akunbeben! Could you mark this PR as ready for review when you've implemented all pending changes?

Yes, I will. 
I'll continue to work on it later today

**Comment by ArtMin96** [comment]
Any news for this PR?

**Comment by akunbeben** [comment]
> DateTimePicker still has the issue with the displayText etc. I'm working on it

Sorry for the delay. After a while, I finally recognized that the problem was when we async load the component, it is not really asynchronous because the livewire.js expects the props in the first place. 

Simply "wire:ignore" along with ax-load="visible" solves the problem. 

Manually tested.

**Comment by danharrin** [comment]
Please revert all unnecessary changes where you are moving large code blocks like this

New imports can be added at the top of the existing code blocks, they don't need to be split

Screenshot 2023-11-15 at 10 03 19

**Comment by akunbeben** [comment]
Changing it, Sorry

File: packages/forms/resources/views/components/date-time-picker.blade.php [hunk 1]
```diff
@@ -57,7 +59,12 @@
         @else
             <div
                 x-ignore
-                ax-load
+                @if (FilamentView::hasSpaMode())
+                    wire:ignore
```
Comment (danharrin) [inline packages/forms/resources/views/components/date-time-picker.blade.php]: Is this wire:ignore deliberate? I think it might break some dynamic features

Comment (danharrin) [reply packages/forms/resources/views/components/date-time-picker.blade.php]: Removed it

**Comment by ArtMin96** [comment]
The problems of Date and Time pickers have not been fixed. Adding `wire:ignore` fixed the issue and it's not breaking the dynamic feature (tested locally).
```blade
@if (FilamentView::hasSpaMode())
    ax-load="visible"
    wire:ignore
@else
    ax-load
@endif
```
image

**Comment by danharrin** [comment]
wire:ignore would break dynamic min/max dates and disabled dates

**Pull request merged**